health-check/services/redis.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * name : redis.js
+ * author : Mallanagouda R Biradar
+ * created-date : 30-June-2025
+ * Description : Redis health check functionality.
+ */
+const { createClient } = require('redis');
+
+exports.check = async (url) => {
```

Comment:
@MallanagoudaB There will be a README.md inside health-check folder. update it to include details added by you